datm/cime_config/stream_definition_datm.xml

```diff
@@ -267,6 +267,44 @@
     <stream_readmode>single</stream_readmode>
   </stream_entry>
 
+
+  <!-- ===================================  -->
+  <!-- datm_mode NEON.$NEONSITE.PRECIP           -->
```

Comment:
As I mention above, I think this should be

NEON.$NMEONSITE.PRISM_PRECIP

Reply:
based on our conversation today would the be NEON.PRISM_PRECIP.$NEONSITE?

Reply:
The order could be done either way with $NEONSITE 2nd or 3rd. It just needs to be consistent. If we think of the order as most general to most specific, I think you are right it should be:

NEON.PRISM_PRECIP.$NEONSITE